### ISSUE TITLE
Videos don't playback on first attempt

### ISSUE THREAD

**Issue by Woolba** [CastagnaIT/plugin.video.netflix#376]
#### Your Environment
- Netflix add-on version: 0.16.0
- Operating system version/name: Fedora 31 (also reproducible on my Nvidia Shield TV)

Used Operating system:
* [ ] Android
* [ ] iOS
 [*] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

### Describe the bug

Whenever I want to watch a video, I have to play the video twice. The first attempt at playing the video, the loading circle will appear for ~10-20 seconds, and then nothing will happen. Pressing the video again will make the loading circle appear, and the video will start playing within a few seconds. This happens on every movie or tv show, and will always play on the second attempt. No error messages are displayed. (note: I am in Australia, using the Australian Netflix without any VPN or other network modifying tools).

#### Expected behavior

The video will load and then playback on the first attempt.

#### Actual behavior

Have to wait for the video to fail to load on the first play attempt, so that it will play on the second play attempt.

#### Steps to reproduce the behavior

1. Navigate to a movie or show in the Netflix addon
2. Press on the video to play it
3. Watch as it loads for ~10-20 seconds and then fails to playback
4. Press on the video a second time to start actual playback

### Debug log

The debug log can be found from this link: https://paste.kodi.tv/ulujoyucun.kodi

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source

**Comment by CastagnaIT** [comment]
Why you have enabled IPC OVER HTTP?

**Comment by Woolba** [comment]
IPC OVER HTTP is enabled to remove the AddonSignals Timeout errors. Disabling it results in the same issue.

**Comment by CastagnaIT** [comment]
disable it and put a new log

**Comment by Woolba** [comment]
Here's a log with ipc over http disabled: https://paste.kodi.tv/avijumuril.kodi

**Comment by CastagnaIT** [comment]
your inputstream adaptive addon is old first update it to latest

**Comment by Woolba** [comment]
It's a little harder to update inputstream adaptive on Linux distributions since it's not provided by the kodi repo (and the release page doesn't seem to have a typical addon zip file that can easily be installed). I'd have to compile it myself. As a quick alternative I've got a log file from my Nvidia Shield TV which has the exact same issue. It is using inputstream adaptive 2.4.2, which is the latest version for Leia according to the github page release versions. It is also running Kodi 18.5, whereas the one on my Linux desktop is 18.4. Here is the log file: https://paste.kodi.tv/cemaceriza.kodi

If this won't do and you'd like a log file from the Linux machine just let me know and I'll take a deeper dive into compiling inputstream adaptive for my operating system. Though it might take a little bit longer to get you the desired log file.

**Comment by CastagnaIT** [comment]
On android uninstall the addon with also user data when prompt
and only after install this
[plugin.video.netflix_0.16.0_20191212.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/3955066/plugin.video.netflix_0.16.0_20191212.zip)
let me know

**Comment by Woolba** [comment]
Unfortunately the same issue persists.
New log: https://paste.kodi.tv/namecabatu.kodi

**Comment by CastagnaIT** [comment]
seem a internet connection problem or you have a problem with dns
Request took 8.673264s
almost 9 seconds to get profiles data from nflx server it is very slow

**Comment by Woolba** [comment]
Wow. I decided to tether my mobile to my PC to test whether or not it'd make a difference. Night and day. Everything loads near instantly and video playback starts on the first attempt. I'm baffled because my mobile and home network both use the same ISP. I don't know why I didn't test this earlier. I think I'm going to have do a closer inspection of my home network and see what's going wrong here.

I'm going to close this issue because it seems to be most definitely an issue somewhere on my end. Sorry and thanks for your time, CastagnaIT.
